### ISSUE TITLE
Emoji form is covering text message field

### ISSUE THREAD

**Issue by vix95** [nextcloud/talk-android#1900]
## Steps to reproduce
1. Open a conversation
2. Type text message
3. Click emoji icon

### Expected behaviour
Emoji form should not cover the text message field.

### Actual behaviour
Emoji form is covering text message field.
![1](https://user-images.githubusercontent.com/38139321/161855422-351d61ba-b4d6-447b-ad4a-4cdfab7dff59.jpg)
![2](https://user-images.githubusercontent.com/38139321/161855429-979f13da-b82b-4922-9ecf-e43ec0be58a0.jpg)

## Device information

**Device:** Samsung Galaxy S21 5G

**Android version:** 12, One UI 4.1

**Talk version:** 13.1.0 Alpha 17

**Comment by AndyScherzinger** [comment]
@mahibi works fine for me on latest master, Android 12, OP9Pro, so this might be OEM specific or related to screen size or density.

**Comment by mahibi** [comment]
works for me on Samsung Galaxy S20FE and also other phones.

does this also happen when you typed only one line of text instead of two?
is your system wide text size in android settings somehow changed?

**Comment by Valdnet** [comment]
On my smartphone it happens again that the window for typing a message with the buttons for sending and changing the keyboard to Emoji is not displayed.
I noticed that it happened twice when receiving a phone call via GSM while using Talk.

![obraz](https://user-images.githubusercontent.com/47037905/161967665-e498f68a-51f0-4195-8dbf-cbad213e88b7.png)

Apps:
Android 10
Talk Nextcloud 13.1.0 Alpha 18

**Comment by vix95** [comment]
> does this also happen when you typed only one line of text instead of two?
> is your system wide text size in android settings somehow changed?

No, it's covering always, it doesn't matter if this is one line, two or three. Always is covered.
No, this is default settings. I have enabled gestures, so I can't see typical icons on bottom.

![image](https://user-images.githubusercontent.com/38139321/162044728-5af756d6-f51c-4153-a9c9-684adc068084.png)

**Comment by jakobroehrl** [comment]
Same problem with Samsung S21 FE
emoji hides input field

**Comment by mahibi** [comment]
just tried with gestures like @vix95 but on my redmi note 7 with android 10 everything is fine with gestures and emoji keyboard.

**Comment by mahibi** [comment]
it is a known bug in the emoji keyboard lib we use. see issue https://github.com/vanniktech/Emoji/issues/758
it makes sense to post any additional infos about this bug over there

**Comment by Valdnet** [comment]
The same problem occurs when a transferred picture in a message, for example, is opened. Emoji are not hidden when you return to the conversation.

**Comment by hoangchungk53qx1** [comment]
the same issues. Is there any way to fix this?
